tooling/dep2.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Why do you need this script?

`yarn pack` should do it automatically for you https://yarnpkg.com/features/workspaces#publishing-workspaces